### ISSUE TITLE
Slight change to herbarium protocol

### ISSUE THREAD

**Issue by PipBrewer** [NHMDenmark/Herbarium-Sheets-workstation#160]
Current herbarium protocol:
1.1.	Two different species on one sheet: If just one of them matches the folder info then treat as normal specimen otherwise contact collections manager. 
1.1.1.	Barcode: corresponding to label information
1.1.2.	Flag as: corresponding to label information + check off the ‘’damaged specimen” box 
1.1.3.	Number of barcodes: only give barcode to the correct species.  

The issue with this protocol is that it is deviating from our overall approach, which is that when there are two obvious specimens on a sheet – they each get a barcode number. Digitisers will not necessarily know whether the two specimens are still currently thought to be different species (sometimes specimen labels are not updated) and they shouldn’t need to be making those kind of decisions during mass digitisation. We shouldn’t be contacting collection managers during digitisation (i.e., stopping the flow), unless it is urgent (e.g., live pests). Things like this can be dealt with later. It is not really a damaged specimen and shouldn’t be marked as such. Finally, this should be marked as a MSO.

In conclusion, it is suggested that this simply be treated as a MSO.
The specimen labels will be transcribed at a later date and the differing species information on the sheets for the two specimens can be highlighted in monthly reports to the collection managers, whoi can decide how (or if) they wish to act without it pausing digitisation. 

This needs mentioning at next herbarium monthly catch up. If agreed, the guides then need updating.

**Comment by RebekkaML** [comment]
this has been implemented with the latest version of the Herbarium guides: https://github.com/NHMDenmark/Herbarium-Sheets-workstation/issues/128.
